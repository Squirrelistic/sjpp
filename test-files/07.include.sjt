{
    "hello": {{json .top}},
    {{include "07.include/include01.sjt"}},
    {{include "07.include/include02.sjt"}}
}