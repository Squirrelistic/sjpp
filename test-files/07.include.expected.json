{
    "hello": "top",
    "included01": "include01",
    "included02": "include02",
    "included0201": "include0201"
}